tests/vmi_networking_test.go

```diff
@@ -721,43 +721,46 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			serverVMI, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(serverVMI.Name, &v13.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
 			Expect(len(serverVMI.Status.Interfaces)).To(Equal(1))
-
-			serverIP := serverVMI.Status.Interfaces[0].IP
-			if netutils.IsIPv6String(serverIP) {
-				By("Checking traceroute from vmi to cluster nodes gateway")
-				// Cluster nodes subnet (docker network gateway)
-				// Docker network subnet cidr definition https://bit.ly/2wZTgMK
-				err := tests.CheckForTextExpecter(serverVMI, createExpectTraceroute6("2001:db8:1::1"), 30)
-				Expect(err).ToNot(HaveOccurred(), "Failed to traceroute to VMI %s within the given timeout", serverVMI.Name)
-			} else {
-				if ipv4NetworkCIDR == "" {
-					ipv4NetworkCIDR = api.DefaultVMCIDR
-				}
-				By("Checking ping to gateway")
-				ipAddr := gatewayIPFromCIDR(ipv4NetworkCIDR)
-				Expect(tests.PingFromVMConsole(serverVMI, ipAddr)).To(Succeed())
-
-				By("Checking ping to google")
-				Expect(tests.PingFromVMConsole(serverVMI, "8.8.8.8")).To(Succeed())
-				Expect(tests.PingFromVMConsole(clientVMI, "google.com")).To(Succeed())
-			}
+			Expect(serverVMI.Status.Interfaces[0].IPs).NotTo(BeEmpty())
 
 			By("starting a tcp server")
 			tcpPort := "8080"
 			err = tests.CheckForTextExpecter(serverVMI, createExpectStartTcpServer(tcpPort), 30)
 			Expect(err).ToNot(HaveOccurred())
 
-			By("Connecting from the client vm")
-			err = tests.CheckForTextExpecter(clientVMI, createExpectConnectToServer(serverIP, tcpPort, true), 30)
-			Expect(err).ToNot(HaveOccurred())
+			for _, serverIP := range serverVMI.Status.Interfaces[0].IPs {
+				if netutils.IsIPv6String(serverIP) {
+					By("Testing IPv6")
+					By("Checking traceroute (IPv6) from vmi to cluster nodes gateway")
+					// Cluster nodes subnet (docker network gateway)
+					// Docker network subnet cidr definition https://bit.ly/2wZTgMK
```

Comment:
removed the comments, if you want please add it later

Reply:
Why would you do that? we need those comments
It is important to explain to where we ping in IPv6 connectivity test since we dont ping to external web service like in IPv4 connectivity test.

Reply:
link is broken, if you want me to fix the link please tell what is the live link.

Reply:
I thought that you added this link at first sorry,
the link is https://github.com/kubevirt/project-infra/blob/master/github/ci/shared-deployments/files/docker-daemon-mirror.conf#L5
it is ok if you dont want to do it in this PR

Reply:
thx
will add permlink in a commit here

Reply:
thx
will add permlink in a commit here
hmm but permlink is super long, i will just add the name of the file that have the cidr ?

the permlink
https://github.com/kubevirt/project-infra/blob/a27023bbae7a51a0a35c989c7aa6ede8a40bd6b0/github/ci/shared-deployments/files/docker-daemon-mirror.conf#L5

Reply:
Just change the current link with the one I sent, if you dont want to do it now so at least bring back the comments they are important (in this case..)
Dont worry about the lgtm label we will add it again if needed

Reply:
done
i am worried because i don't find that important, in order to block PRs and work. 
but updated since its important to you.